src/pages/home/report/SystemChatReportFooterMessage.tsx

```diff
@@ -86,6 +86,5 @@ export default withOnyx<SystemChatReportFooterMessageProps, SystemChatReportFoot
     },
     activePolicyID: {
         key: ONYXKEYS.NVP_ACTIVE_POLICY_ID,
-        initialValue: null,
```

Comment:
Is this needed?

Reply:
There is no need to set `null` as the `initialValue`, as `null` will be ignored and is only used when setting/merging